planet/http.py

```diff
@@ -413,6 +414,34 @@ async def stream(
         finally:
             await response.aclose()
 
+    def client(self, name: str, base_url: Optional[str] = None) -> object:
+        """Get a client by its module name.
+
+        Parameters:
+            name: the name of the client module: data, orders, or
+                subscriptions.
+
+        Returns:
+            A client instance.
+
+        Raises:
+            ClientError when no such client can be had.
+
+        """
+        # To avoid circular dependency.
+        from planet.clients.data import DataClient
+        from planet.clients.orders import OrdersClient
+        from planet.clients.subscriptions import SubscriptionsClient
+
+        try:
+            return {
+                'data': DataClient,
+                'orders': OrdersClient,
+                'subscriptions': SubscriptionsClient
```

Comment:
For now, I'm statically configuring the mapping between names and clients here. Getting too fancy would be a mistake. We have no plans to have multiple clients per API or support user-supplied client plugins.

Reply:
Looking at my PR as if I was a reviewer, it occurs to me that this static mapping could be moved to `planet/client/__init__.py`. That would let us reduce those 3 imports to 1.